### PR TITLE
Update vn.m3u

### PR DESCRIPTION
**Pull request by HaNoiIPTV**

### CODE REVIEW

**Comment by freearhey** [comment]
@HaNoiIPTV I don't understand, why you decided to change `tvg-id`?

**Comment by Dum4G** [comment]
http://gg.gg/HaNoiIPTV-ANTV-Truyen_hinh_An_Ninh_HD doesn't open in vlc, returns chunklist in a browser

File: channels/vn.m3u [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+#EXTINF:-1 tvg-id="antv.vn" tvg-name="ANTV" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://raw.githubusercontent.com/HaNoiIPTV/HaNoiIPTV.m3u/master/ANTV%20-%20Truy%E1%BB%81n%20h%C3%ACnh%20C%C3%B4ng%20an%20Nh%C3%A2n%20d%C3%A2n%20HD.png" group-title="",ANTV - Truyền hình An Ninh
 http://antvlive.ab5c6921.cdnviet.com/antv/chunklist.m3u8
-#EXTINF:-1 tvg-id="ANTVHD.vn" tvg-name="ANTV HD" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://tduc.tk/logo/antv.png" group-title="",ANTV HD (576p)
-http://antvlive.ab5c6921.cdnviet.com/antv/playlist.m3u8
-#EXTINF:-1 tvg-id="BRT.vn" tvg-name="BRT" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://brt.vn/common/v2/images/Banner150219.png" group-title="",BRT (360p)
+#EXTINF:-1 tvg-id="antv.vn" tvg-name="ANTV HD" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://raw.githubusercontent.com/HaNoiIPTV/HaNoiIPTV.m3u/master/ANTV%20-%20Truy%E1%BB%81n%20h%C3%ACnh%20C%C3%B4ng%20an%20Nh%C3%A2n%20d%C3%A2n%20HD.png" group-title="",ANTV - Truyền hình An Ninh HD (720p)
+http://gg.gg/HaNoiIPTV-ANTV-Truyen_hinh_An_Ninh_HD
```
Comment (freearhey) [inline channels/vn.m3u:5]: https://github.com/iptv-org/iptv/pull/2914#issuecomment-849584583

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

File: channels/vn.m3u (hunk 1 above)
Comment (themak1985) [reply channels/vn.m3u:5]: Nothing wrong with http://antvlive.ab5c6921.cdnviet.com/antv/playlist.m3u8
New link (gg.gg) redirects to https://livecdn.fptplay.net/sdc/anninhtv_hls.smil/chunklist_b2500000.m3u8

Comment (Dum4G) [reply channels/vn.m3u:5]: Not really. Both links do work in SimpleTV

**Comment by freearhey** [comment]
Closing due to lack of any action.

**Pull request closed without merging**